scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc

```diff
@@ -39,6 +39,12 @@ include::modules/ztp-acm-adding-images-to-mirror-registry.adoc[leveloffset=+1]
 
 include::modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#enable-cim[Enabling the Central Infrastructure Management service]
```

Comment:
I don't know that I understand why this is being added both to the procedure and as an additional resource to the procedure/module. I'd consider omitting one.